### PR TITLE
File Added:- Assignment 1 and Assignment 2

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

**Review by Pawan1005 — APPROVED** [review]

File: Shubham_Dalal/JavaScript/Assignment2/Assignment_two_Q2.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <script>
+            function fun() {
+                first = parseInt(document.getElementById("input1").value);
+                second = parseInt(document.getElementById("input2").value);
+                third = parseInt(document.getElementById("input3").value);
+                fourth = parseInt(document.getElementById("input4").value);
+                fifth = parseInt(document.getElementById("input5").value);
+
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment2/Assignment_two_Q2.html]: remove extra line

File: Shubham_Dalal/JavaScript/Assignment2/Assignment_two_Q2.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                total = first + second + third + fourth + fifth;
+                percentage = total / 5;
+
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment2/Assignment_two_Q2.html]: remove extra line

File: Shubham_Dalal/JavaScript/Assignment2/Assignment_two_Q2.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                border-radius: 20px 30px 50px;
+                margin-top: 100px;
+                padding: 10px;
+            }
+            input {
+                width: auto;
+                padding: 10px;
+                border-radius: 10px;
+                margin: 10px;
+                /* align-self: center; */
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment2/Assignment_two_Q2.html]: remove comment

File: Shubham_Dalal/JavaScript/Assignment2/Assignment_two_Q2.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            }
+
+            button {
+                margin-top: 20px;
+                padding: 10px;
+                /* width: fit-content; */
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment2/Assignment_two_Q2.html]: remove comments

File: Shubham_Dalal/JavaScript/Assignment2/Assignment2_Q1.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
+.flexbox{
```
Comment (nileshp1122) [inline Shubham_Dalal/JavaScript/Assignment2/Assignment2_Q1.css:1]: before parenthesis space required

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Pull request merged**